v2.1/admin-ui-hardware-dashboard.md

```diff
@@ -0,0 +1,109 @@
+{{site.data.alerts.end}}
+
+## System CPU Percent
+
+<img src="{{ 'images/v2.1/admin_ui_system_cpu.png' | relative_url }}" alt="CockroachDB Admin UI System CPU Percent graph" style="border:1px solid #eee;max-width:100%" />
+
+- In the node view, the graph shows the percentage of the total CPU seconds per second used by the system calls made by CockroachDB for the selected node.
+
+- In the cluster view, the graph shows the percentage of the total CPU seconds per second used by the system calls made by CockroachDB across all nodes.
+
+{{site.data.alerts.callout_info}}
+For multi-core systems, the system CPU Percent can be greater than 100%. Full utilization of one core is considered as 100% CPU usage. If you have n cores, then the system CPU Percent can range from 0% (indicating an idle system) to (n*100)% (indicating full utilization).
+{{site.data.alerts.end}}
+
+## Memory Usage
+
+<img src="{{ 'images/v2.1/admin_ui_memory_usage_new.png' | relative_url }}" alt="CockroachDB Admin UI Memory Usage graph" style="border:1px solid #eee;max-width:100%" />
+
+- In the node view, the graph shows the memory in use by CockroachDB for the selected node.
+
+- In the cluster view, the graph shows the memory in use by CockroachDB across all nodes in the cluster.
+
+## Disk Read Bytes
+
+<img src="{{ 'images/v2.1/admin_ui_disk_read_bytes.png' | relative_url }}" alt="CockroachDB Admin UI Disk Read Bytes graph" style="border:1px solid #eee;max-width:100%" />
+
+- In the node view, the graph shows the current moving average, over the last 10 seconds, of the number of bytes read per second by all processes including CockroachDB for the selected node.
+
+- In the cluster view, the graph shows the current moving average, over the last 10 seconds, of the number of bytes read per second by all processes including CockroachDB across all nodes.
+
+## Disk Write Bytes
+
+<img src="{{ 'images/v2.1/admin_ui_disk_write_bytes.png' | relative_url }}" alt="CockroachDB Admin UI Disk Write Bytes graph" style="border:1px solid #eee;max-width:100%" />
+
+- In the node view, the graph shows the current moving average, over the last 10 seconds, of the number of bytes written per second by all processes including CockroachDB for the node.
+
+- In the cluster view, the graph shows the current moving average, over the last 10 seconds, of the number of bytes written per second by all processes including CockroachDB across all nodes.
+
+## Disk IOPS in Progress
+
+<img src="{{ 'images/v2.1/admin_ui_disk_iops.png' | relative_url }}" alt="CockroachDB Admin UI Disk IOPS in Progress graph" style="border:1px solid #eee;max-width:100%" />
+
+- In the node view, the graph shows the number of disk reads and writes in queue for all processes including CockroachDB for the selected node.
+
+- In the cluster view, the graph shows the number of disk reads and writes in queue for all processes including CockroachDB across all nodes in the cluster.
```

Comment:
forgot to mention — this will always show up as 0 on Mac, because the library we're using doesn't report it on Mac, only Linux and maybe Windows. (There may be some way to get it; I just haven't found it yet)

Reply:
Filed https://github.com/cockroachdb/cockroach/issues/27927; can use as known limitation